src/components/filtered-product-list/FilteredProductList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ProductCard from "../product-card/ProductCard.jsx";
+import filterByField from "../../helpers/filter-by-field-function/FilterByField.js";
+
+
+
+const FilteredProductList = ({ meals, activeCategory, visibleCount, updateBasketCount }) => {
+    const filteredMeals = filterByField(meals, 'category', activeCategory);
+    return(
+        <>
+            <div className='productsGridContainer'>
+                {filteredMeals.slice(0, visibleCount).map((meal) => (
+                    <ProductCard
+                        key={meal.id}
+                        product={{
+                            id: meal.id,
```

Comment:
There is no need to map a field if the key does not change, could just spread the object and then map the changes